xbmc/pictures/GUIWindowPictures.cpp

```diff
@@ -502,6 +502,8 @@ void CGUIWindowPictures::GetContextButtons(int itemNumber, CContextButtons &butt
   CGUIMediaWindow::GetContextButtons(itemNumber, buttons);
   if (item && !item->GetProperty("pluginreplacecontextitems").asBoolean())
     buttons.Add(CONTEXT_BUTTON_SETTINGS, 5);                  // Settings
+
+  BaseContextMenuManager::Get().AppendVisibleContextItems(item, buttons);
```

Comment:
If you mean setting default/user ordering, sure it could work. But that requires refactoring the whole context button lists. For now I suggest keeping things simple, so either just add them once in `CGUIMediaWindow` or add it in every window like now. (PS: the "manage" menu is a completely separate menu for the video library so it doesn't transfer that easily over)